setup/security/ssl.md

```diff
@@ -313,27 +367,42 @@ If you have problems...
 * If you are running Spinnaker in a distributed environment, have you run
 `hal deploy connect`?
 
-## About network configurations
-
-Each authentication mechanism is configured differently depending on where the
-SSL connection terminates.
-
-### Server-terminated SSL
 
-Terminating SSL within the Gate server is the de-facto way to enable SSL for
-Spinnaker. This works with or without a load balancer proxying traffic to this
-instance.
+## Using a custom CA for internal communications
+There are a lot of places in Spinnaker which support the ability to configure custom Java trust/key stores for 
+organizations who use internally signed certificates. In some cases, however, this isn’t supported yet but you still 
+need to talk to a service which serves one of these certificates. This post will show you how to import your 
+certificate into a Java trust/key store and configure a Spinnaker service with it.
 
-![SSL terminated at server through load balancer](/setup/security/authentication/network-arch/server-ssl-termination.png)
+Create a temporary copy of your system’s Java trust/key store and import your internal certificate. If you’re on a Mac,
+this will be located at `/usr/libexec/java_home/)/jre/lib/security/cacerts`. It will be different on Linux.
+```
+mkdir /tmp/custom-trust-store`
+cp {path-to-cacerts} /tmp/custom-trust-store
+keytool import -alias custom-ca -keystore /tmp/custom-trust-store/cacerts -file {your-internal-certificate}
+```
 
-### Load Balancer-terminated SSL
+The below example instructions apply when use kubernetes to deploy spinnaker.  If not using spinnaker, you'll have 
+to get the cacerts file updated as appropriate for your environment.  
+```bash
+kubectl create secret generic -n {your-spinnaker-namespace} internal-trust-store \
+   --from-file /tmp/custom-trust-store/cacerts
+```
+Configure a Spinnaker service with the new trust/key store using a volume mount. In this example we’ll be configuring 
+Front50 with this new store.
 
-A common practice is to offload SSL-related bits to outside of the server in
-question. This is fully supported in Spinnaker, but it does affect the
-authentication configuration slightly. See your [authentication
-method](/setup/security/authentication/) for specifics.
+In `~/.hal/default/service-settings/front50.yml`
+```
+kubernetes:
+  volumes:
+  - id: internal-trust-store
+    mountPath: /etc/ssl/certs/java
+    type: secret
+```
 
-![SSL terminated at load balancer](/setup/security/authentication/network-arch/lb-ssl-termination.png)
+Redeploy Spinnaker using `hal deploy apply`.
+The Spinnaker component for which you configured the volume mount should now be using the new trust/key store by 
```

Comment:
Maybe say "Front50 should now be using..." (since we state that the example will be configuring Front50)?